src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HttpLoggingServiceCollectionExtensions.cs

```diff
@@ -1,72 +1,110 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if NET8_0_OR_GREATER
+
 using System;
+using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.AspNetCore.Diagnostics.Logging;
+using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.Extensions.Compliance.Classification;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Http.Diagnostics;
-using Microsoft.Extensions.Options;
-using Microsoft.IO;
+using Microsoft.Shared.DiagnosticIds;
 using Microsoft.Shared.Diagnostics;
 
 namespace Microsoft.Extensions.DependencyInjection;
 
 /// <summary>
 /// Extension methods to register the HTTP logging feature within the service.
 /// </summary>
+[Experimental(diagnosticId: Experiments.HttpLogging, UrlFormat = Experiments.UrlFormat)]
 public static class HttpLoggingServiceCollectionExtensions
 {
     /// <summary>
-    /// Adds components for incoming HTTP requests logging into <see cref="IServiceCollection"/>.
+    /// Enables redaction of HTTP logging.
     /// </summary>
-    /// <param name="services">The <see cref="IServiceCollection"/> to add the service to.</param>
-    /// <returns>The value of <paramref name="services"/>.</returns>
+    /// <param name="services">The service collection.</param>
+    /// <param name="configureRedaction">Configures the redaction options.</param>
+    /// <returns>The original service collection.</returns>
```

Comment:
I've updated this text for everything in dotnet/extensions to say:

```
<returns>The value of <paramref name="services".</returns>
```

Can we do the same here?